### DISCUSSION TITLE
Shuffle ControlNet and repositories (SDXL)

### DISCUSSION THREAD

**Discussion by Darkflo264** [Comfy-Org/ComfyUI#1360] [Q&A]
Hi all!

I have read about the filename check for a shuffle controlnet in commit 65cae62, but as for now i was not able to find a shuffle ControlNet for SDXL anywhere.
Does anyone have a source for this one?

Also, i´m a bit lost of keeping a overview of all available ControlNet for SDXL. 
Do we have (i have not found one yet) a discussion or info about all currently available ControlNets for SDXL and where to get them?

Thank you in advance!

**Comment by Zaferturan** [comment]
https://civitai.com/models/136070?modelVersionId=267621

**Reply by HTPAUN** [reply]
不行 报错
